specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Indeed, now that we have sample CVRs from CDOS, this definition could be enormously enriched, though we usually stick to modeling only exactly those subterms that are necessary to define the system's behavior.  I'll make the change that you note, but we should realize that it'll likely evolve more soon.